modules/users/tests/server/user.server.model.tests.js

```diff
@@ -436,17 +440,18 @@ describe('User Model Unit Tests:', function () {
     });
 
     it('should allow valid email address - "abc-def@abc.com"', function (done) {
-      var _user = new User(user);
+      var _user1 = new User(user1);
 
-      _user.email = 'abc-def@abc.com';
-      _user.save(function (err) {
+      _user1.email = 'abc-def@abc.com';
+      _user1.save(function (err) {
         if (!err) {
-          _user.remove(function (err_remove) {
+          _user1.remove(function (err_remove) {
+            should.not.exist(err);
```

Comment:
@lirantal I see the same thing as @rhutchison with line 449

`should.not.exist(err);` will always pass, since it couldn't have gotten into this block otherwise from line 447. I think you could move the assertion at line 449 to 447; right after the `_user1.save`